apps/src/music/utils/MusicMetrics.ts

```diff
@@ -37,3 +47,12 @@ function decorateMessage(message: string | object): object {
     lab: 'music',
   };
 }
+
+function getDimensions(): MetricDimension[] {
+  return [
+    {
+      name: 'Lab',
+      value: 'music',
+    },
```

Comment:
Oh, will we want to differentiate based on which sound library is being loaded, particularly later when we have more than one sound library in broad use?

Reply:
Oh good call!